lib/src/test/resources/generators/play-2-standalone-json-spec-quality.txt

```diff
@@ -301,7 +291,7 @@ package com.gilt.quality.v0.models {
     private[this]
     val byName = all.map(x => x.toString.toLowerCase -> x).toMap
 
-    def apply(value: String): ExternalServiceName = fromString(value).getOrElse(UNDEFINED(value))
+    def apply(value: String): ExternalServiceName = fromString(value).get
```

Comment:
Maybe throw an exception with a nice error message instead of using `get` directly? `get` could throw `NoSuchElementException` which isn't very helpful in my opinion.